hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/DatanodeAdminDefaultMonitor.java

```diff
@@ -201,6 +201,7 @@ private void check() {
         iterkey).iterator();
     final List<DatanodeDescriptor> toRemove = new ArrayList<>();
     final List<DatanodeDescriptor> unhealthyDns = new ArrayList<>();
+    boolean inValidState = false;
```

Comment:
IMO the variable name is confusing in the first glance. Is it "valid" or "invalid" when it's true? How about renaming it as "isValid"?

Reply:
Thanks @jojochuang - I have made the required changes.